.github/workflows/deploy-devnet.yml

```diff
@@ -36,6 +38,10 @@ jobs:
           memory_mb: 128MB
           region: ${{ secrets.SMV_DEVNET_GCP_REGION }}
           credentials: ${{ secrets.SMV_GCP_SA_KEY }}
-          env_vars: MONGO_SECRET_NAME="${{ env.MONGO_SECRET_NAME }}",TWITTER_SECRET_NAME="${{ env.TWITTER_SECRET_NAME }}",TWITTER_SEARCH_TEXT="${{ env.TWITTER_SEARCH_TEXT }}",TWITTER_REPLY_SUCCESS="${{ env.TWITTER_REPLY_SUCCESS }}",TWITTER_REPLY_INVALID_FORMAT="${{ env.TWITTER_REPLY_INVALID_FORMAT }}",TWITTER_REPLY_INVALID_SIGNATURE="${{ env.TWITTER_REPLY_INVALID_SIGNATURE }}"
+          env_vars: MONGO_SECRET_NAME=${{ env.MONGO_SECRET_NAME }},TWITTER_SECRET_NAME=${{ env.TWITTER_SECRET_NAME }},TWITTER_SEARCH_TEXT=${{ env.TWITTER_SEARCH_TEXT }},TWITTER_REPLY_SUCCESS=${{ env.TWITTER_REPLY_SUCCESS }},TWITTER_REPLY_INVALID_FORMAT=${{ env.TWITTER_REPLY_INVALID_FORMAT }},TWITTER_REPLY_INVALID_SIGNATURE=${{ env.TWITTER_REPLY_INVALID_SIGNATURE }}
```

Comment:
@fkondej Should these be `{{ secrets.XX }}` instead of `{{ env.XX }}` ?

- `MONGO_SECRET_NAME=${{ env.MONGO_SECRET_NAME }}` 
- `TWITTER_SECRET_NAME=${{ env.TWITTER_SECRET_NAME }}`